sinonjs/fake-timers
# Rename package to @sinonjs/fake-clock

This PR renames the package to `@sinonjs/fake-clock`.

#### Purpose

It was suggested that we rename this package to have a less confusing name (I can't recall who suggested it, or in which communication channel).

I will soon be doing a larger effort in improving the documentation of Sinon and associated libraries. I thought it would be beneficial to that effort to get the rename of packages out of the way first.

See https://github.com/sinonjs/sinon/issues/1651

#### Plan

1. Rename repository
1. Merge this branch
1. Bump the major version
1. Publish the renamed package to `npm` registry
1. Deprecate the `lolex` package in the `npm` registry
1. Update `sinon` to use the new package

#### How to verify

1. Check out this branch
1. (this repo): `npm ci`
1. (this repo): `npm link`
1. (sinon repo): `npm link @sinonjs/fake-clock`
1. (sinon repo): Update the `lib/sinon/util/fake-timers.js` to use `@sinonjs/fake-clock` instead of `lolex`
1. (sinon repo): `npm test`
1. Observe tests pass
